appsync-bedrock/README.md

```diff
@@ -0,0 +1,77 @@
+# AWS AppSync to Amazon Bedrock
+
+This pattern shows how to invoke Amazon Bedrock models from AppSync through HTTP resolvers.
+
+Why? This pattern can be used as a starting point to leverage AWS AppSync pipeline resolvers and Amazon Bedrock. In your pipeline resolver, you could for example first fetch some information from another datasource (external API, database, ...) and use the information returned by this datasource to include it in your prompts when invoking Amazon Bedrock models. With pipeline resolvers, you do not need to call (and therefore pay for) compute resources (such as lambda functions) to perform this logic.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/appsync-bedrock
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
```

Comment:
it would be great to mention that you need to get permissions to use bedrock models from your account before proceeding.

Reply:
Done